package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.6.0",
-    "react-dom": "^16.6.0",
+    "@material-ui/core": "^3.4.0",
+    "@material-ui/icons": "^3.0.1",
+    "mdbreact": "^4.8.2",
```

Comment:
Not sure that this library is acceptable in the scope of this task. In the description to the task, there is clearly defined two options:

react-bootstrap or material-UI

So, please remove it.